ios/FluentUI/Button/ButtonTokenSet.swift

```diff
@@ -0,0 +1,258 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+// MARK: ButtonStyle
+
+@objc(MSFButtonStyle)
+public enum ButtonStyle: Int, CaseIterable {
+    case primaryFilled
+    case primaryOutline
+    case dangerFilled
+    case dangerOutline
+    case secondaryOutline
+    case tertiaryOutline
+    case borderless
+}
+
+/// Design token set for the `Button` control.
+public class ButtonTokenSet: ControlTokenSet<ButtonTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// Defines the background color of the button
+        case backgroundColor
+
+        /// Defines the border color of the button
+        case borderColor
+
+        /// Defines the size of the border around the button
+        case borderSize
```

Comment:
Nit: `borderWidth` or `strokeWidth` to better align with upcoming Fluent 2 token value names?